cmd/test/main.go

```diff
@@ -515,3 +519,22 @@ func wereViolationsFound(validationManager *ValidationManager, results *evaluati
 		return false
 	}
 }
+
+func saveDefaultRulesAsFile(ctx *TestCommandContext, preRunPolicies string) {
+	if preRunPolicies == "" {
+		return
+	}
+
+	homeDir, err := os.UserHomeDir()
+	if err != nil {
+		return
+	}
+
+	defaultRulesFilePath := filepath.Join(homeDir, ".datree", "defaultRules.yaml")
+
+	writeErr := ioutil.WriteFile(defaultRulesFilePath, []byte(preRunPolicies), 0644)
```

Comment:
what does 0644 mean? can you put it in a constant for readability?

Reply:
Correct